seqr/fixtures/reference_data.json

```diff
@@ -1081,7 +1081,7 @@
     "model": "reference_data.omim",
     "pk": 2,
     "fields": {
-        "gene": 3,
+        "gene": 48,
```

Comment:
changing the gene associated with this condition means we can test cases where a duplicate  phenotype mim number is associated with a discovery variant